test/ctr.bats

```diff
@@ -711,3 +711,46 @@ function teardown() {
 	cleanup_pods
 	stop_crio
 }
+
+@test "ctr /etc/resolv.conf rw/ro mode" {
+	start_crio
+	run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+	writeconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["command"] = ["sh", "-c", "echo test >> /etc/resolv.conf"]; json.dump(obj, sys.stdout)')
```

Comment:
I don't mind having multiple config files. It does make it easier to look at them and debug using crioctl directly.